app/components/article-content.js

```diff
@@ -388,6 +390,59 @@ export default Component.extend(
 			}
 		},
 
+		/**
+		 * Handles opening sections when onclick event occurs on reference
+		 *
+		 * @returns {void}
+		 */
+		handleRefs() {
+			const citeNotes = this.element.querySelectorAll(
+					'ol[class*="references"] ' +
+					'li[id^="cite_note"] ' +
+					'a[href*="cite_ref"]'
+				),
+				citeRefs = this.element.querySelectorAll('a[href*="cite_note"]'),
+				referencesList = this.element.querySelectorAll('ol[class*="references"]');
+
+			let citeNotesHeader = 0;
+
+			if (referencesList.length) {
+				const referencesListHead = referencesList[0].closest('.mobile-hidden');
+				if (referencesListHead) {
+					citeNotesHeader = referencesListHead.previousElementSibling;
+				}
+			}
+
+			if (citeNotes.length && citeRefs.length) {
+
+				// If referenced section is closed, open it.
+				citeNotes.forEach((element) => {
+					element.onclick = function () {
+						const currentCiteNote = document.getElementById(element.hash.slice(1, element.length));
+						const closest = currentCiteNote.closest('.mobile-hidden');
+
+						if (closest) {
+							let currentHeader = document.getElementById(closest.previousElementSibling.id);
+							if (currentHeader.className !== 'open-section') {
+								currentHeader.click();
```

Comment:
although this works 
it's rather ugly
you could extract body of the click handler of a header
and call it directly